src/runner.ts

```diff
@@ -148,7 +148,7 @@ export class ResolveRunner implements Types.IResolveRunner {
 
         if (!r.resolved.result) continue;
 
-        this._source = produce(this._source, draft => {
+        this._source = produce(this._source, (draft: any) => {
```

Comment:
Vincenzo resorting to any :scream:

Reply:
It was any even before, but now with Immer 3.0 is not wrapped anymore in stuff. At least, we can now shame ourselves.